src/identity/IdentityPage.test.tsx

```diff
@@ -71,25 +86,34 @@ describe('Identity page', () => {
 
     it('lets you switch to the tests tab', async () => {
       await wait(() => expect(screen.queryByText(/Mock QRCode: mock-sharing-code/i)).toBeTruthy());
-      expect(screen.queryByText(/tests will be shown here/i)).toBeFalsy();
+      expect(screen.queryByText(/no tests yet/i)).toBeFalsy();
       fireEvent.click(screen.getByText(/tests/i));
-      await wait(() => expect(screen.queryByText(/tests will be shown here/i)).toBeTruthy());
+      await wait(() => expect(screen.queryByText(/no tests yet/i)).toBeTruthy());
       expect(screen.queryByText(/Mock QRCode: mock-sharing-code/i)).toBeFalsy();
       fireEvent.click(screen.getByText(/profile/i));
       await wait(() => expect(screen.queryByText(/Mock QRCode: mock-sharing-code/i)).toBeTruthy());
-      expect(screen.queryByText(/tests will be shown here/i)).toBeFalsy();
+      expect(screen.queryByText(/no tests yet/i)).toBeFalsy();
+    });
+
+    it('shows all your tests on the tests tab', async () => {
+      await wait(() => expect(screen.queryByText(/first middle last/i)).toBeTruthy());
```

Comment:
first middle last?  Seems an odd expectation for this test?

Reply:
Honestly this is just waiting for the page to load.